### ISSUE TITLE
[VueLoaderPlugin Error] No matching use for vue-loader is found.

### ISSUE THREAD

**Issue by tomrijnbeek** [JeffreyWay/laravel-mix#1867]
-   Laravel Mix Version: 4.0.7 (`npm list --depth=0`)
-   Node Version (`node -v`): v8.9.4
-   NPM Version (`npm -v`): v6.5.0
-   OS: macOS Mojave

### Description:

When I try to run Laravel Mix, it complains that the Vue loader cannot find something to be applied to. This is somewhat expected, since we don't use Vue ourselves.

This is the top part of the stacktrace:

```
Error: [VueLoaderPlugin Error] No matching use for vue-loader is found.
Make sure the rule matching .vue files include vue-loader in its use.
    at VueLoaderPlugin.apply (/Users/Tom/dev/web/dominating12/node_modules/vue-loader/lib/plugin.js:58:13)
```

### Steps To Reproduce:

Our webpack.min.js looks like this:

```
let mix = require('laravel-mix');

mix.sass('resources/assets/sass/style.scss', 'public/assets/css')
   .sass('resources/assets/sass/error.scss', 'public/assets/css')
   .coffee('resources/assets/coffee/app/**/*.coffee', 'public/assets/js/app.coffee.js')
   .ts('resources/assets/ts/app/main.ts', 'public/assets/js/bundle.js')
   .extract(['jquery'])
   .sourceMaps();

if (mix.inProduction()) {
    mix.version();
}
```

I run `npm run dev` to start the compilation. The relevant scripts in package.json are:

```
    "dev": "npm run development",
    "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
```

It looks like the problem is that both the Coffee and the TypeScript components get the `webpackConfig()` methods called, which call down to the Vue `webpackConfig()` method, which installs the `VueLoaderPlugin` plugin. The plugin is never used, and we subsequently get an error about that being the case.

I suppose a quick fix would be to add an empty vue file somewhere, but that's just a hack to make the compiler happy. I find it somewhat strange that the script is so opinionated about using Vue, especially since this is a migration from an older Elixir config. This issue is blocking us from completing that migration.

**Comment by JeffreyWay** [comment]
This should be fixed in 4.0.8, which I just tagged. If you update and it's still not working, please let me know. :)

Thanks!

**Comment by tomrijnbeek** [comment]
Oh wow such a quick turnaround. That seems to have fixed it, thanks so much!
